src/context.rs

```diff
@@ -526,3 +510,20 @@ impl Display for ActorShutdown {
         f.write_str("Actor is shutting down")
     }
 }
+
+#[cfg(feature = "metrics")]
+const ACTOR_LABEL: &str = "actor";
+
+#[cfg(feature = "metrics")]
+const MESSAGE_LABEL: &str = "message";
+
+#[cfg(feature = "metrics")]
+lazy_static::lazy_static! {
+    static ref PROCESSING_DURATION_HISTOGRAM: prometheus::HistogramVec = prometheus::register_histogram_vec!(
+        "xtra_message_processing_duration_seconds",
+        "The processing time of an xtra message in seconds.",
+        &[ACTOR_LABEL, MESSAGE_LABEL],
+        vec![0.0000001, 0.000001, 0.000002, 0.000005, 0.00001, 0.0001, 0.001, 0.01, 0.02, 0.05, 0.1, 0.2, 0.5, 1.0, 2.0, 5.0, 10.0, 20.0, 50.0, 100.0]
```

Comment:
I'm a bit scared of the sheer amount of metrics we will create with this. 
There are 20 buckets for each unique actor times their messages: did you check how many metrics that will be for us?

Reply:
I didn't count them but can have a look.

We can always disable them once we learned what is useful and what isn't but there needs to be room for experimenting :)

Also, isn't more metrics a really small price to pay in exchange for the insight on how well individual parts of our system are working?

Reply:
Not if it kills our setup :D

Reply:
What numbers are we talking where that actually happens?